en/docs/concepts/oidc/intro-oidc.md

```diff
@@ -41,7 +41,7 @@ The exact flow differs based on the authentication flow or the grant type.  Howe
 
 The following steps explain how this protocol works in a simple manner.
 
-  ![oidc-flow](../assets/img/concepts/oidc-basic-flow.png)
+  ![oidc-flow](../../assets/img/concepts/oidc-basic-flow.png)
```

Comment:
this doesn't need double slash...having "../../assets" instead of "../assets" causes a warning on the build log

Reply:
As I have included the .md file inside a new directory "../../" is needed. Otherwise it wont display the image.